src/main/java/duke/Events.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+/**
+ * A task type that the chatting bot can create.
+ */
+public class Events extends Task {
```

Comment:
Shouldn't the class name here be `Event` as the plural form would indicate a collection of Event objects?